### PR TITLE
Ensure that the notification icon and journald logging plugin prefs windows are not destroyed when they recieve a delete event

### PR DESCRIPTION
**Pull request by JasonLG1979**

### CODE REVIEW

File: pithos/plugins/journald_logging.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def __init__(self, parent, settings):
         self.pithos_window = parent
         self.settings = settings
         self.set_resizable(False)
-        self.connect('response', self._on_response)
+        self.connect('delete-event', lambda *ignore: self.do_response(Gtk.ResponseType.CANCEL))
```
Comment (TingPing) [inline pithos/plugins/journald_logging.py]: Probably just want `self.response()`

Comment (JasonLG1979) [reply pithos/plugins/journald_logging.py]: OK

Comment (JasonLG1979) [reply pithos/plugins/journald_logging.py]: Well i guess I din't understand what you mean?
```self.connect('delete-event', lambda *ignore: self.do_response(self.response())```

Comment (TingPing) [reply pithos/plugins/journald_logging.py]:
```python
self.connect('delete-event', lambda *ignore: self.response(Gtk.ResponseType.CANCEL))
 ```

Comment (JasonLG1979) [reply pithos/plugins/journald_logging.py]: right on

Comment (JasonLG1979) [reply pithos/plugins/journald_logging.py]: Nope that didn't work. I got:

```gtk_container_foreach: assertion 'GTK_IS_CONTAINER (container)' failed``

When I tried to reopen the window and a segfault and crash when I clicked on it.

Comment (JasonLG1979) [reply pithos/plugins/journald_logging.py]: @TingPing the problem I'd guess is that ```self.response``` doesn't return True.

Comment (JasonLG1979) [reply pithos/plugins/journald_logging.py]: ```self.connect('delete-event', lambda *ignore: self.response(Gtk.ResponseType.CANCEL) or True)``` seems to work though.

**Comment by JasonLG1979** [comment]
There what do you think of that?

**Comment by JasonLG1979** [comment]
There you go @TingPing all rebased against master ready to merge :)

**Pull request merged**